src/views/nunjucks/macros/pluralise.njk

```diff
@@ -0,0 +1,3 @@
+{% macro pluralise(collection, singular, plural) %}
```

Comment:
In my latest PR I've also created a plural implementation using a Nunjucks filter - perhaps we can have a quick look at this together